testing/powerpc-utils/APKBUILD

```
@@ -0,0 +1,37 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=powerpc-utils
+pkgver=1.3.3
+pkgrel=0
+pkgdesc="Utilities which are intended for maintenance of powerpc platforms"
+url="https://github.com/nfont/powerpc-utils"
+arch="all"
+license="GPL"
+depends=""
+makedepends="autoconf automake"
+install=
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/nfont/powerpc-utils/archive/v$pkgver.tar.gz"
+
+_builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd "$_builddir"
+	./autogen.sh
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--mandir=/usr/share/man \
+		--without-librtas \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$_builddir"
+	make DESTDIR="$pkgdir" install
+	return 0
```

This is wrong, why you want the package to silently pass failed `make install`?